src/Dialect/ONNX/ONNXOpsHelper.cpp

```diff
@@ -342,6 +342,21 @@ bool AreTheSameAxisArray(int64_t rank, ArrayAttr lhsAttr, ArrayAttr rhsAttr) {
   return true;
 }
 
+/// Convert ConstantOp to ArrayAttr and test if they have the same values
+bool AreTheSameConstantOpDenseAttr(
+    Builder &builder, int64_t rank, Value lhsOp, Value rhsOp) {
+  ONNXConstantOp lhsConstOp = dyn_cast<ONNXConstantOp>(lhsOp.getDefiningOp());
+  ONNXConstantOp rhsConstOp = dyn_cast<ONNXConstantOp>(rhsOp.getDefiningOp());
+  if ((lhsConstOp && rhsConstOp) &&
+      (!isFromNone(lhsConstOp) && !isFromNone(rhsConstOp))) {
```

Comment:
Why 'isFromNone' is needed here? Should this function return true if both are `isFromNone`?

Reply:
Sorry, I was originally testing if `%cst = constant unit` was considered a ConstantOp and if so, test if it's also None. But the cast returns a null anyway, so I forgot to take that out. I will do so in my next push.